learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,159 @@
+# Faceted search
+
+Meilisearch filters can be used to build faceted search interfaces. This type of interface allows users to refine search results based on broad categories or facets. Faceted search provides users with a quick way to narrow down search results by selecting categories relevant to what they are looking for. A faceted navigation system is an **intuitive interface to display and navigate through content**. Facets are used in the UI as filters that users can apply to refine the results in real-time.
+
+Done right, it’s like allowing users to find their way as they would in a physical store. Not only would users be able to find what they’re looking for, they can also see other relevant products you have. If your user is looking for a blanket, facets would ask questions like: What color do you want? Do you have a particular brand in mind? Any preferences for materials? How much are you willing to pay?
+
+Facets are common in ecommerce sites like Amazon. When users perform a search, they are presented not only with a list of results but also with a list of facets which you can see on the sidebar in the image below:
+
+![Meilisearch demo for an ecommerce website displaying faceting UI](/faceted-search/facets-ecommerce.png)
+
+Faceted search interfaces often have a count of how many results belong to each facet. This gives users a visual clue of the range of results available for each facet.
+
+### Filters or facets
+
+In Meilisearch, facets are a specific use-case of filters. The question of whether something is a filter or a facet is mostly one pertaining to UX and UI design.
+
+Meilisearch does not differentiate between facets and filters. This means that, despite its name, facets can be used with any attributes added to `filterableAttributes`.
+
+## Configuring and using facets
+
+Like any other filter, attributes you want to use as facets must be added to the [`filterableAttributes`](/reference/api/settings.md#filterable-attributes) list in the index's settings before they can be used. Once they have been configured, you can search for facets with [the `facets` search parameter](/reference/api/search.md#facets).
+
+::: warning
+Synonyms don't apply to facets. Meaning, if you have `SF` and `San Francisco` set as synonyms, faceting by `SF` and `San Francisco` will show you different results.
+:::
+
+Suppose you have an ecommerce dataset called `ecommerce` containing the following fields:
+
+```json
+{
+  "id": "711decb2a3fdcbbe44755afc5af25e2f",
+  "title": "Kitchenex Stainless Steel Flatware Pie Server and Pie Cutter Set of 2",
+  "description": "| The Kitchenex Stainless Pie Server|…",
+  "url": "https://www.amazon.com/dp/B07D2LJFYW",
+  "images": [
+    "https://images-na.ssl-images-amazon.com/images/I/41AeR2oo75L._.jpg"
+  ],
+  "category": "Home & Kitchen",
+  "brand": "Dr. Pet",
+  "price": 16.84,
+  "rating": 4.7,
+  "reviews_count": 7,
+}
```

Comment:
I recommend simplifying this sample document:
- only show the minimum amount of fields required to establish the document as a webshop product;
- shorten the existing fields. The smaller they are, the easier it is for a reader to process and remember them;
- this one is arguable, but I would add a couple of documents to this. Like a minimal sample dataset. When I was writing the geosearch guide, I found it very helpful to have a small pool of data I could refer to across the text (e.g. "if you do this, then you will see productA and productB, but if you do that you will only see productC").

To illustrate:

```js
[
  {
    id: 1,
    title: "Leather jacket" ,
    brand: "Nike",
    category: "sportswear",
    price: 17.14
  },
  {
    id: 2,
    title: "Leather shoes" ,
    brand: "Adidas",
    category: "sportswear",
    price: 1.14
  },
]
```